### PR TITLE
CPS-33: Fix to mask sensitive output generated by any resource of typ…

### PR DESCRIPTION
**Pull request by SeemaGC-Procore**
…e 'random_*'

### CODE REVIEW

File: procore_terraform_wrapper/procore_terraform_wrapper.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def reduce_plan(input: str) -> str:
         input = match.expand(r"\1 State refresh removed for brevity\n\2")
     else:
         logger.warning('Failed to match regex for terraform init replacement. Please report this to the Atlantis maintainers as a bug!')
+        input = re.sub(r"\"random_*.+?}", repl="State refresh removed to mask sensitive output\n", string=input, count=0, flags=re.MULTILINE|re.DOTALL)
```
Comment (mralanlee) [inline procore_terraform_wrapper/procore_terraform_wrapper.py]: remove one tab

**Review by mralanlee — CHANGES_REQUESTED** [review]

**Review by mralanlee — APPROVED** [review]
lgtm

**Comment by SeemaGC-Procore** [comment]
merging to master

**Comment by SeemaGC-Procore** [comment]
Merging to master

**Pull request merged**